src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import duke.Task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**Deadline class for task with deadlines
+ */
+public class Deadline extends Task {
+
+    protected String by;
+    protected String transformedDate;
+
+    /**
+     * Constructor for deadline class
```

Comment:
For Javadoc comments, perhaps you can attempt:
1. Punctuation behind each parameter description.
2. Empty line between description and parameter section